src/test/java/com/google/api/codegen/testdata/discogapic/java/java_fakebilling.v1.json.baseline

```diff
@@ -548,6 +553,16 @@ public final class BillingAccountsGetHttpRequest {
       return this;
     }
 
+    public List<String> getFooRepeated() {
+      return fooRepeated;
+    }
+
+    public Builder setFooRepeated(List<String> fooRepeated) {
+      this.fooRepeated = new LinkedList<>();
```

Comment:
We should use an ImmutableList instead of LinkedList so that the message is immutable after construction.